docs/src/services.md

```diff
@@ -13,55 +6,109 @@ eoAPI combines several state-of-the-art projects to create a full Earth Observat
 - STAC Items And Mosaic Raster Tiles API built on top of https://github.com/stac-utils/titiler-pgstac
 - OGC Features and Vector Tiles API built on top of https://github.com/developmentseed/tipg
 
-## Metadata
+## Database
 
-Built on [stac-fastapi.pgstac](https://github.com/stac-utils/stac-fastapi) application, adding a **`TiTilerExtension`** and a simple **`Search Viewer`**.
+The STAC database is at the heart of eoAPI and is the only **mandatory** service. We use [`PgSTAC`](https://github.com/stac-utils/pgstac) Postgres schema and functions which provides functionality for STAC Filters, CQL2 search, and utilities to help manage the indexing and partitioning of STAC Collections and Items.
 
-The service includes:
+> PgSTAC is used in production to scale to hundreds of millions of STAC items. PgSTAC implements core data models and functions to provide a STAC API from a PostgreSQL database. PgSTAC is entirely within the database and does not provide an HTTP-facing API. The Stac FastAPI PgSTAC backend and Franklin can be used to expose a PgSTAC catalog. Integrating PgSTAC with any other language with PostgreSQL drivers is also possible.
+>
+> PgSTAC Documentation: https://stac-utils.github.io/pgstac/pgstac
+>
+> pyPgSTAC Documentation: https://stac-utils.github.io/pgstac/pypgstac
 
-- Full **stac-fastapi** implementation - see [docs](http://localhost:8081/docs) if using the `docker-compose` configuration.
-- Simple STAC Search **viewer** - see [viewer](http://localhost:8081/index.html) if using the `docker-compose` configuration.
-- **Proxy** to the tiler endpoint for STAC Items.
-    
-When the `TITILER_ENDPOINT` environment variable is set (pointing to the `raster` application), additional endpoints will be added to the stac-fastapi application (see: [stac/extension.py](https://github.com/developmentseed/eoAPI/blob/main/src/eoapi/stac/eoapi/stac/extension.py)):
+## Metadata
+
+The Metadata service deployed in eoAPI is built on [stac-fastapi.pgstac](https://github.com/stac-utils/stac-fastapi) application.
+
+By default, the STAC metadata service will have a set of endpoint to *search* and list STAC collections and items.
 
-- `/collections/{collectionId}/items/{itemId}/tilejson.json`: Return the `raster` tilejson for an item
-- `/collections/{collectionId}/items/{itemId}/viewer`: Redirect to the `raster` viewer
 
 <p align="center">
-<img src="https://user-images.githubusercontent.com/10407788/146790933-e439893c-ef2e-4d78-a372-f2f18694836c.png"/>
-<p align="center">STAC Search viewer</p>
+<img alt="stac-fastapi" src="https://github.com/developmentseed/eoAPI/assets/10407788/d0963386-1c8f-4607-98b8-3b0edb341a5e"/>
 </p>
 
-Code: [/runtime/eoapi/stac](https://github.com/developmentseed/eoAPI/tree/main/runtime/eoapi/stac)
+!!! example
+
+    - https://stac.eoapi.dev landing page
```

Comment:
adding some links to serve as example of endpoints that will be deployed by default